class ValidException(Exception):
    pass


class RepoException(Exception):
    pass
